web/styles/popovers.css

```diff
@@ -414,9 +414,14 @@ ul {
         }
     }
 
+    .user_profile_edit_button {
+        margin-left: 10px;
+        width: 25px;
+        text-align: center;
+    }
+
     #edit-button {
```

Comment:
Can you do a second commit renaming this to `user_profile_edit_button_icon`? `#edit-button` is too vague of a selector name.

Reply:
> Can you do a second commit renaming this to `user_profile_edit_button_icon`? `#edit-button` is too vague of a selector name.

Done.